doc/nrf/known_issues.rst

```diff
@@ -303,6 +303,25 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
+.. rst-class:: v1-6-0
+
+KRKNWK-8211: The ``ZB_ZDO_SIGNAL_LEAVE`` signal is generated twice during ZC factory reset
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9714: Device association fail if the Request Key packet is retransmitted
+  If the Request Key for the TCLK is retransmitted and the coordinator sends two, different new keys, it confuses the joiner's logic and lead to unsuccessful key verification.
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9743 Timer can't be stopped in Zigbee routers and coordinators
+  The call to the ``zb_timer_enable_stop()`` API has no effect on the timer logic in Zigbee routers and coordinators.
+
+.. rst-class:: v1-6-0
+
+KRKNWK-10490: Dead lock in NCP frame fragmentation logic
+  If the last piece of a fragmented NCP command is not delivered, the receiving side becomes unresponsive to further commands.
```

Comment:
```suggestion
KRKNWK-10490: Deadlock in the NCP frame fragmentation logic
  If the last piece of a fragmented NCP command is not delivered, the receiving side becomes unresponsive to commands.
```
No workaround, right?

Reply:
Yes, there is no workaround for this issue.